docs/generic.md

```diff
@@ -50,8 +51,8 @@ where 'JSON input' is
   // path to the package (relative to the --source directory)
   // defaults to "."
   "path": ".",
-  // option to specify lockfile path, must be an absolute path if specified
-  // defaults to "artifacts.lock.yaml", relative to path
+  // option to specify lockfile path: absolute or relative to "path"
```

Comment:
nitpick: instead of just "relative to path" (was confusing at first glance without further code context), say "relative to package path"